examples/experimental/Length of Stay demo on MIMIC/example/test_trainer.py

```diff
@@ -0,0 +1,100 @@
+"""module containing functions showing how to use the other functions in the
+    Length of Stay demo on MIMIC folder
+"""
+import os
+import sys
+import json
+import numpy as np
+import logging
+import torch.nn as nn
+from pprint import pformat
+from torch import randint as torch_randint
+from torch import float as torch_float
+from torch.utils.data import TensorDataset
+from torch.utils.data import DataLoader
+from torch.utils.data.sampler import SubsetRandomSampler
+
+print(os.path.dirname(os.path.dirname(__file__)))
+CONFIG_PATH = os.path.join(
+    os.path.dirname(os.path.dirname(__file__)), "metadata", "config_result_paper.json",
+)
+
+
+def test_trainer() -> None:
+    """function showing how to use the train function
+    """
+    sys.path.append(".")
+    os.chdir(os.path.dirname(os.path.dirname(__file__)))
```

Comment:
Why do you need this? the  CONFIG_PATH used below should be absolute and shouldn't require this, right?

Reply:
These settings allow you to import the PyTorch module by launching the script from anywhere. It's ok for you ?